queries/cpp/highlights.scm

```diff
@@ -189,3 +189,6 @@
   ["<" ">"] @punctuation.bracket)
 
 (literal_suffix) @operator
+
+((comment) @comment.documentation
+  (#lua-match? @comment.documentation "^/[*][*][^*].*[*]/$"))
```

Comment:
This should be in C.

Reply:
It doesn't seem to be an "official" standard whereas c++ does consider it